content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -847,6 +847,41 @@ not vulnerable to ordering changes in the list.
 
 Once the last finalizer is removed, the resource is actually removed from etcd.
 
+### Force deletion
+
+{{< feature-state feature_gate_name="AllowUnsafeMalformedObjectDeletion" >}}
+
+{{< caution >}}
+This may break the workload associated with the resource being force deleted, if it
+relies on the normal deletion flow, so cluster breaking consequences may apply.
+{{< /caution >}}
+
+By enabling the delete option `ignoreStoreReadErrorWithClusterBreakingPotential`, the
+user can perform an unsafe force **delete** operation of an undecryptable/corrupt
+resource. This option is behind an ALPHA feature gate, and it is disabled by
+default. In order to use this option, the cluster operator must enable the feature by
+setting the command line option `--feature-gates=AllowUnsafeMalformedObjectDeletion=true`.
+
+{{< note >}}
+In addition to `delete` rights, the user performing the force **delete**
+operation must have `unsafe-delete-ignore-read-errors` rights on the given resource.
```

Comment:
(style nit)
```suggestion
The user performing the force **delete** operation must be permitted to perform
both the **delete** and **unsafe-delete-ignore-read-errors** verbs on that resource.
```

We write API verbs in non-monospaced bold. I assume that **unsafe-delete-ignore-read-errors** is a verb and that a SubjectAccessReview happens. Have I got that right?

Also, avoid implying that clusters always use Kubernetes RBAC. Many do but it's not mandatory to.